apps/frontend/src/features/register/devcon/Register.tsx

```diff
@@ -194,8 +135,7 @@ const RegisterDevcon: React.FC<RegisterDevconProps> = ({ savedTap }) => {
     try {
       // Register user
       await registerUser({
-        signinToken: code,
-        email,
+        email: username,
```

Comment:
I bet this is where Response.email is coming from (so ignore my earlier comment).